src/components/Map/index.tsx

```diff
@@ -49,17 +50,21 @@ class MyMap extends Component<MapPropsInterface, MyMapParams> {
 
     let prevPosition: { lat?: string, lon?: string } = {};
 
-    try {
-      prevPosition = JSON.parse(window.localStorage.getItem('viewport') || '');
-    } catch (err) {
-      console.log(err);
+    const { lat, lon } = this.props.match.params;
+
+    if (!lat && !lon) {
+      try {
+        prevPosition = JSON.parse(window.localStorage.getItem('viewport') || '');
+      } catch (err) {
+        console.log(err);
```

Comment:
я бы так сделал, чтобы красненькое было :)

на твой вкус

```suggestion
        console.error(err);
```